app/src/main/java/com/teamagam/gimelgimel/app/map/viewModel/IMapView.java

```diff
@@ -0,0 +1,16 @@
+package com.teamagam.gimelgimel.app.map.viewModel;
+
+import com.teamagam.gimelgimel.app.map.model.VectorLayer;
+import com.teamagam.gimelgimel.app.map.model.geometries.PointGeometry;
+
+/**
+ * Created on 8/18/2016.
+ * TODO: complete text
+ */
+public interface IMapView {
+    void takePicture();
```

Comment:
What would the map view take care about taking picture?

Something else should implement this method (i.e. External class to take picture from user)